sickbeard/search.py

```diff
@@ -579,11 +579,16 @@ def searchProviders(show, episodes, manualSearch=False, downCurQuality=False, ma
 
         # Update the cache if a manual search is being runned
         if manual_snatch:
-            for searched_episode in episodes:
-                results = curProvider.cache.update_cache_manual_search(searchResults[searched_episode.episode])
-                if results is not None:
-                    # If we have at least a result from one provider, it's good enough to be marked as result
-                    manual_snatch_results.append(results)
+            # Let's create a list with episodes that we where looking for
+            searched_episode_list = [episode_obj.episode for episode_obj in episodes]
+            # Add the -1 to also match season pack results
+            searched_episode_list.append(-1)
+            episodes_search_result = [x for x, _ in searchResults.items()]
```

Comment:
I'm creating a list of the Keys for the search results. The Keys are the mapped episodes or -1